lib/src/quick_usb_desktop.dart

```diff
@@ -329,4 +329,12 @@ class _QuickUsbDesktop extends QuickUsbPlatform {
     }
     return result;
   }
+
+  @override
+  Future<void> setAutoDetachKernelDriver(bool enable) async {
+    assert(_devHandle != null, 'Device not open');
+    if (Platform.isLinux) {
+      _libusb.libusb_set_auto_detach_kernel_driver(_devHandle!, enable ? 1 : 0);
+    }
```

Comment:
```dart
    var setConfig = _libusb.libusb_set_configuration(_devHandle!, config.id);
    if (setConfig != libusb_error.LIBUSB_SUCCESS) {
      print('setConfig error: ${_libusb.describeError(setConfig)}');
      return false;
    }
    return true;
```

Reply:
Care to explain?
Please see original comment why this is void.

Reply:
We'd better expose the result of any possible call to _libusb. Maybe someone would care someday

Reply:
Any possible result in this case is always LIBUSB_SUCCESS.

Reply:
No problem. Could you rebase onto the master branch?